.github/workflows/release.yml

```diff
@@ -97,6 +97,16 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: startsWith(matrix.os, 'ubuntu')
+      - name: Upload release assets (linux AppImage)
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ needs.release.outputs.upload-url }}
+          asset_path: dist/Alephium-0.10.2.AppImage
```

Comment:
replace `0.10.2` with `${{ needs.release.outputs.version }}` and need

Reply:
oh nice catche, it's done :+1: